### DISCUSSION TITLE
Could you make GUI to select ROI in video

### DISCUSSION THREAD

**Discussion by vuongmanh15** [SWHL/RapidVideOCR#41] [Ideas]
Could you make GUI to select ROI in video, example in attach file.
Thank for amazing project!!

![image](https://github.com/SWHL/RapidVideOCR/assets/36305735/2bb0d5eb-0317-478c-b718-a6d070ed284f)

**Comment by SWHL** [comment]
This part is done with VideoSubFinder software. You can directly download the VideoSubFinder to select ROI.
VideoSubFinder: https://sourceforge.net/projects/videosubfinder/

**Reply by vuongmanh15** [reply]
Oh, thanks so much
